js/utility.js

```diff
@@ -26,6 +26,22 @@ const Utils = {
     return console.error(`No window available to get the current url!`);
   },
 
+  getCurrentURLQuery() {
```

Comment:
any reason we don't just use https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams?

Reply:
mostly that when last I checked, that wasn't broadly supported, and I never bothered to check if that had changed in the mean time! Good call =)